src/asm/parser.y

```diff
@@ -1164,6 +1166,14 @@ incbin:
 	}
 ;
 
+include_once:
```

Comment:
Move this below the `include` rule so `incbin` doesn't separate them.

Reply:
The `include_once` grammar rule has been moved above `incbin`